src/index.html

```diff
@@ -1,13 +1,381 @@
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights
+              integrated into
+              the
+              frame give you always-on visibility day and night.
+            </div>
+          </section>
+          <section class="second_section">
+            <div class="image_preview">
+              <div class="image_over">
+                <div class="base_image block_3_1_image"></div>
+              </div>
+              <div class="image_over">
+                <div class="base_image block_3_2_image"></div>
+              </div>
+            </div>
+            <div class="bike_section_header bike_section_margin">
+              Hydraulic disc brakes & Lightweight
+            </div>
+            <div class="bike_section_description additional_description">
+              Brakes with total stopping power the second you make contact. The removable battery has up
+              to 70km
+              battery
+              autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on
+              visibility day
+              and
+              night.
+            </div>
+          </section>
+
+          <button class="base_button text_button" type="button">
+            Explore
+          </button>
+        </div>
+      </section>
+      <section
+        class="main_content main_content_gap"
+        id="contacts"
+      >
+        <h1 class="header_section_base header_section_second">
+          Contact us
+        </h1>
+        <div class="base_section_content contact_section_gap">
+          <form
+            class="form_contact"
+            method="post"
+            onsubmit="this.reset(); return false;"
+          >
+            <div class="form_field">
+              <input
+                class="input_base_form input_form_text"
+                name="name"
+                placeholder="Name"
+                required
+                type="text"
+              >
+              <input
+                class="input_base_form input_form_text"
+                name="email"
+                placeholder="Email"
+                required
+                type="email"
+              >
+              <textarea
+                class="input_base_form input_form_area"
+                name="message"
+                placeholder="Message"
+                required
+              ></textarea>
+            </div>
+            <button class="base_button text_button submit_button" type="submit">
+              Send
+            </button>
+          </form>
+          <section class="contact_info_base contact_info_section">
+            <section class="contact_info_base contact_info_block">
+              <div class="contact_header contact_base">
+                Phone
+              </div>
+              <a
+                class="contact_info contact_base"
+                href="tel:+1 234 5555-55-55"
```

Comment:
Need fit format phone everywhere only in href
```suggestion
                href="tel:+123455555555"
```